src/zope/dublincore/browser/metadataedit.py

```diff
@@ -13,39 +13,45 @@
 ##############################################################################
 """Dublin Core Meta Data View
 """
-__docformat__ = 'restructuredtext'
-
 from datetime import datetime
 from zope.event import notify
 from zope.dublincore.interfaces import IZopeDublinCore
 from zope.lifecycleevent import ObjectModifiedEvent, Attributes
 from zope.i18nmessageid import MessageFactory
+
 _ = MessageFactory('zope')
 
+try:
+    unicode
+except NameError:
+    unicode = str  # PY3
```

Comment:
This looks like an actual bugfix for an actual bug on Python 3.  Worth a changelog note, I think.  (Ideally also a unit test but.)

Reply:
You are right, this code is completely untested.

Reply:
I added a test and a change log entry in #9.